_maps/RandomRuins/LavaRuins/lavaland_surface_colossus_arena.dmm

```diff
@@ -0,0 +1,28 @@
+"a" = (/turf/closed/indestructible{desc = "An extremely sturdy wall made of frigid metal, engraved with strange markings."; icon = 'icons/turf/walls/cult_wall.dmi'; icon_state = "cult"; name = "runed wall"},/area/ruin/powered)
```

Comment:
Nooooooo. If you want indestructible cult walls, make a subtype of indestructible walls that actually smooths, don't edit the turfs.